wiki/People/Elite_Nominators/it.md

```diff
@@ -0,0 +1,104 @@
+# Elite Nominators
+
+Gli **Elite Nominators** sono dei membri del [Beatmap Nominator Team](/wiki/People/Beatmap_Nominators) e [Nomination Assessment Team](/wiki/People/Nomination_Assessment_Team) che si sono distinti dimostrando un enorme dedizione al [processo di classificazione](/wiki/Beatmap_ranking_procedure) e l'ambiente globale della community del [mapping](/wiki/Beatmapping), spiccando come i migliori nominator dei loro tempi.
+
+Gli Elite Nominators sono selezionati dal Nomination Assessment Team in base a molteplici fattori, tra cui (ma non limitati a):
+
+- Numero di [nomine](/wiki/Beatmap_ranking_procedure#nominations)
+- Qualità delle beatmap nominate
+- Qualità del [modding](/wiki/Modding)
+- Quantità e [gravità](/wiki/People/Nomination_Assessment_Team/SEV_rating) dei [ripristini delle nomination](/wiki/Beatmap_ranking_procedure#nomination-resets) ricevuti
```

Comment:
```suggestion
- Quantità e [gravità](/wiki/People/Nomination_Assessment_Team/SEV_rating) dei [ripristini delle nomine](/wiki/Beatmap_ranking_procedure#nomination-resets) ricevuti
```

![immagine](https://github.com/ppy/osu-wiki/assets/5209299/6511a6ab-f63c-405c-bef6-6cc206bc6bab)
On the website it is translated as "nomine", and translating to ripristini delle nomine could make more sense to the reader